src/node_modules/logger.js

```diff
@@ -1,54 +1,51 @@
-const $cannotConnect = Symbol('cannotConnect property');
-const $connection = Symbol('connection property');
-const $console = Symbol('console property');
-const $failedAttempts = Symbol('failedAttempts property');
-const $flatten = Symbol('flatten property');
-const $flattenArrays = Symbol('flattenArrays property');
-const $getConnection = Symbol('getConnection method');
-const $host = Symbol('host property');
-const $levels = Symbol('levels property');
-const $Logger = Symbol('Logger property (winston / bunyan)');
-const $maxFailedAttempts = Symbol('maxFailedAttempts property');
-const $minLevel = Symbol('minLevel property');
-const $port = Symbol('port property');
-const $replacer = Symbol('replacer property');
-const $secure = Symbol('secure property');
-const $serialize = Symbol('serialize method');
-const $tempLevel = Symbol('tempLevel property (winston)');
-const $timestamp = Symbol('timestamp property');
-const $token = Symbol('token property');
-const $toLevel = Symbol('toLevel method');
-const $withLevel = Symbol('withLevel property');
-const $withStack = Symbol('withStack property');
+const $connection               = Symbol('connection property');
+const $reconnection             = Symbol('reconnection property');
+const $reconnect                = Symbol('reconnect module');
+const $reconnectInitialDelay    = Symbol('reconnectionInitialDelay property');
+const $reconnectMaxDelay        = Symbol('reconnectionMaxDelay property');
+const $reconnectBackoffStrategy = Symbol('reconnectBackoffStrategy property');
+const $bufferFull               = Symbol('bufferFull property');
+const $console                  = Symbol('console property');
+const $flatten                  = Symbol('flatten property');
+const $flattenArrays            = Symbol('flattenArrays property');
+const $getConnection            = Symbol('getConnection method');
+const $host                     = Symbol('host property');
+const $levels                   = Symbol('levels property');
+const $Logger                   = Symbol('Logger property (winston / bunyan)');
+const $minLevel                 = Symbol('minLevel property');
+const $port                     = Symbol('port property');
+const $replacer                 = Symbol('replacer property');
+const $secure                   = Symbol('secure property');
+const $serialize                = Symbol('serialize method');
+const $tempLevel                = Symbol('tempLevel property (winston)');
+const $timeout                  = Symbol('inactivityTimeout property');
+const $timestamp                = Symbol('timestamp property');
+const $token                    = Symbol('token property');
+const $toLevel                  = Symbol('toLevel method');
+const $withLevel                = Symbol('withLevel property');
+const $withStack                = Symbol('withStack property');
 
 // PATTERNS ////////////////////////////////////////////////////////////////////
 
-const newline = /\n/g;
+const newline      = /\n/g;
```

Comment:
This spacing is forced and for to align it with the following assignments. Makes the code more readable.